man/special-symbols.Rd

```diff
@@ -54,6 +57,7 @@ DT[, c(.(y=max(y)), lapply(.SD, min)),
 DT[, grp := .GRP, by=x]                # add a group counter
 DT[, grp_pct := .GRP/.NGRP, by=x]      # add a group "progress" counter
 X[, DT[.BY, y, on="x"], by=x]          # join within each group
+DT[X, on=.NATURAL]                     # join X and DT naturally on common column similar to X[on=Y]
```

Comment:
columns* (also imo 'naturally' isn't required since 'join X and DT on common columns' would indicate a natural join)

Reply:
done 👍